File: problem_solving_for_beginners/array_problems/c-programming/search_element.c
#include<stdio.h>
/**
 * main - search key in element of array. Print "Key found" otherwise "Key not found"
 *
 * Return: 0 Always (success)
 */
int main(void)
{
    int arr[10], n, i, key;
    printf("Enter the size of array: ");
    scanf("%d", &n);

    for(i = 0; i < n; i++)
    {
        printf("Enter the element of array: ");
        scanf("%d", &arr[i]);
    }

    printf("Enter the key: ");
    scanf("%d", &key);
    
    int flag = 0;
    for (i = 0; i < n; i++)
    {
        if (arr[i] == key)
        {
            flag = 1;
            break;
        }
    }
    
    if (flag == 1)
        printf("Key found\n");
    else
        printf("Key not found\n");

    return 0;
}

File: problem_solving_for_beginners/array_problems/python/search_element.py
#!/usr/bin/python3
"""Search a key in elements of array"""
arr = []
n = int(input("Enter a size of array: "))

for i in range(n):
    i = int(input("Enter element of array: "))
    arr.append(i)


key = int(input("Enter a key: "))
flag = 0

for i in range(n):
    if arr[i] == key:
        flag = 1
        break
        
if flag == 1:
    print("Key found")
else:
    print("Key not found")